Source/WebCore/animation/CSSPropertyAnimation.cpp

```diff
@@ -3937,7 +3938,6 @@ CSSPropertyAnimationWrapperMap::CSSPropertyAnimationWrapperMap()
         case CSSPropertyOverscrollBehaviorInline:
         case CSSPropertyOverscrollBehaviorX:
         case CSSPropertyOverscrollBehaviorY:
-        case CSSPropertyPaddingBlock: // logical shorthand
         case CSSPropertyPaddingInline: // logical shorthand
```

Comment:
Seems wrong that this whole change is specific to `padding-block` and applies to no other logical shorthand (`padding-inline`, `margin-block/inline`, etc.)

Reply:
I disagree. We got a bug report specifically about this property and this patch addresses this. Yes, a part of the source change will be beneficial to additional properties of that ilk, but a smaller change is my preference because we’ve gotten regressions in the last doing too much at once. We can add additional properties bit by bit.

Reply:
I think it would make sense to at least do `padding-inline` and `padding-block` at the same time.

Because inconsistency in behavior could be considered as a bug.